examples/Workflow/WorkflowConsoleApp/WorkflowConsoleApp.csproj

```diff
@@ -9,6 +9,7 @@
     <TargetFramework>net6</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <LangVersion>latest</LangVersion>
+    <NoWarn>612,618</NoWarn>
```

Comment:
?

Reply:
The `<NoWarn>612,618</NoWarn>` disables the "obsolete" warnings that we get when using the DaprClient methods. I'm actually not sure that these are needed, though, now that we've reverted to using `WorkflowEngineClient` in this initial release of the sample.

Reply:
These are to supress the warnings that come from the fact that we are using functions that are still in alpha.